audisp-tacplus/audisp-tacplus.c

```diff
@@ -45,19 +45,25 @@
 
 #include <sys/types.h>
 #include <sys/wait.h>
+#include <pwd.h>
```

Comment:
I don't see anything related to sonic. Could you give concrete title? Does it really need to know SONiC?

Why you mentioned sonic in first commit message?